java/test/jmri/jmrit/PackageTest.java

```diff
@@ -26,27 +27,17 @@ static public void main(String[] args) {
     public static Test suite() {
         TestSuite suite = new TestSuite("jmri.jmrit.PackageTest");   // no tests in this class itself
 
-        // this next line causes obscure AWT errors when here,
```

Comment:
This comment refers to a test that is running about five lines down, so it seems to be OBE. If not, it indicates that some test is not cleaning up behind itself correctly.